src/vs/workbench/contrib/files/electron-sandbox/fileActions.contribution.ts

```diff
@@ -71,6 +71,12 @@ MenuRegistry.appendMenuItem(MenuId.OpenEditorsContext, {
 	command: revealInOsCommand,
 	when: REVEAL_IN_OS_WHEN_CONTEXT
 });
+MenuRegistry.appendMenuItem(MenuId.OpenEditorsContextShare, {
```

Comment:
Fyi the "Open Editors" view is hidden by default.

Reply:
:cool: I'd still like Share to appear for Open Editors if the view does happen to be visible (it is one of several places where Share should IMO appear).